uk.org.greenend.chiark.sgtatham.puzzles.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: uk.org.greenend.chiark.sgtatham.puzzles 
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME platform?

Reply:
It's a GTK app, is there a more basic runtime I could use?

Reply:
freedesktop-sdk

Reply:
`org.freedesktop.Platform` and matching `*.Sdk`. version 22.08